builtin/providers/google/resource_compute_global_address.go

```diff
@@ -14,7 +14,12 @@ func resourceComputeGlobalAddress() *schema.Resource {
 		Create: resourceComputeGlobalAddressCreate,
 		Read:   resourceComputeGlobalAddressRead,
 		Delete: resourceComputeGlobalAddressDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Same as above: I tried actually using this to import a resource after making the change and the "name" field wasn't filled out so this copies ID to name.